src/framework/ui/service/common.service.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Text } from 'react-native';
+/**
+ * Creates array from `source` parameter if needed
+ *
+ * @param source (T | T[]) - any object which should be wrapped to an array
+ * @returns (T[]) - wrapped `source` if was need to wrap, `source` otherwise
+ */
+
+export function toArray<T>(source: T | T[]): T[] {
+  if (Array.isArray(source)) {
+    return source;
+  }
+  return [source];
+}
+
+export function toReactElement(source: React.ReactChild): React.ReactElement<any> {
```

Comment:
We probably don't need this

Reply:
used [here](https://github.com/akveo/react-native-ui-kitten/blob/4b3fcca3d3f0adfd36209a78180f70769295c9c5/src/framework/ui/viewPager/viewPager.component.tsx#L57)